test/user.env.example

```diff
@@ -0,0 +1,10 @@
+export INSTANCE_NAME_PREFIX=colemick
+export LOCATION=westus2
```

Comment:
Maybe, but it's sort of become standard, I see it used in a loooot of places - I'd prefer to keep with LOCATION for consistency's sake.